wdl-grammar/tests/parsing/prefix-exprs/source.wdl

```diff
@@ -0,0 +1,11 @@
+# This is a test of prefix expressions.
+
+version 1.1
+
+task test {
+    Boolean a = !true
+    Boolean b = !!!!!!!!!false
+    Int c = -2
+    Float d = -2.0
+    Int e = ---------42
```

Comment:
Is this allowed? (I haven't checked the spec) I would hope/expect an error will be thrown for this at some stage.

Reply:
It should be, it's just a series of negation expressions that are right-associative, no different from an explicit parenthesized expression of `-(-42)`, same [as in Rust](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=6b6ed2a2ac3075e63933fc0ee7366cd0).

Reply:
Huh. Learned something new. IDK why, but I was working under the assumption that `-42` is the same as `--42` is the same as `---42` and so on. But in hindsight that doesn't make much sense. Ignore this then!